src/styles/blocks/todolist.scss

```diff
@@ -0,0 +1,143 @@
+.todolist {
+  margin: 0;
+  padding: 0;
+
+  list-style: none;
+
+  &__item {
+    position: relative;
+
+    border-bottom: 1px solid #ededed;
+
+    font-size: 24px;
+
+    &:last-child {
+      border-bottom: none;
+    }
+
+    &--editing {
+      padding: 0;
+
+      border-bottom: none;
+
+      &:last-child {
+        margin-bottom: -1px;
+      }
+    }
+
+    &--completed label {
+      color: #d9d9d9;
+      text-decoration: line-through;
+    }
+
+    & label {
+      display: block;
+
+      padding: 15px 15px 15px 60px;
+      transition: color 0.4s;
+
+      word-break: break-all;
+      line-height: 1.2;
+    }
+  }
+
+  &__edit-field {
+    display: none;
+
+    &:focus {
+      outline: 0;
+    }
+  }
+
+  &__item--editing &__edit-field {
+    display: block;
+
+    padding: 12px 16px;
+    margin: 0 0 0 $editFieldeftMargin;
+    width: 467px;
+    box-sizing: border-box;
+    border: 1px solid #999;
+    box-shadow: inset 0 -1px 5px 0 rgba(0, 0, 0, 0.2);
+
+    font: "Helvetica Neue", Helvetica, Arial, sans-serif;
+    font-size: 24px;
+    font-family: inherit;
+    font-weight: inherit;
+    line-height: 1.4em;
+    -webkit-font-smoothing: antialiased;
+    color: inherit;
+
+    @media screen and (max-width: 550px) {
+      & {
+        width: calc(100vw - 2 * #{$appSideMargin} - #{$editFieldeftMargin});
```

Comment:
Replace the '$appSideMargin' with a valid SCSS variable, it seems to be a typo.

Reply:
Didn't catch. As for me, this is correct. I just made a typo in "$editFieldeftMargin".